Oleksadr_Petrov/HW8/Task2.py

```diff
@@ -0,0 +1,13 @@
+def custom_filter(callback: callable, sequence: iter):
+    return [value for value in sequence if value == callback(value)]
+
+
+def even_numbers(number: int):
+    if number % 2 == 0:
+        return number
```

Comment:
you can use lambda except for writing your own function. Just a suggestion, no need to remove it

Reply:
the previous solution remained in the file, but additionally added an example with a lambda expression